translatorbot/Dockerfile

```diff
@@ -0,0 +1,40 @@
+# setup
+FROM python:3.11.5
```

Comment:
Just to confirm: is this a remnant from the prior demo or is this Dockerfile needed here? Wondering because I see some references to the talk to LLM demo below with things like grandma images.